app/views/admin/configs/index.haml

```diff
@@ -11,6 +11,7 @@
       = f.input :eto_api_available, label: 'Is the ETO API available?'
       = f.input :healthcare_available, label: 'Enable Healthcare section?'
       = f.input :project_type_override, label: 'Override ALL project types?', hint: 'Treat the Project Type Override as a global override instead of just an override for federal reporting.'
+      = f.input :release_duration, collection: ["Indefinite", "One Year"]
```

Comment:
Can you move the collection here up to the Config model? It could be `def self.available_release_durations`